main_fr.tex

```diff
@@ -0,0 +1,21 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+% Language setting
+\usepackage[french]{babel}
+
+\def\sections{sections}
+
+\newcommand{\version}{
+  Version 1.0
+}
+
+\newcommand{\intro}{
+  This is a community-driven project, which has a \href{https://github.com/Heegu-sama/Homm3BG}{GitHub repository}.
+  Everyone is welcome to contribute, make changes, and fix errors.
+  If you simply want to leave feedback, please do so in the original \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{BoardGameGeek thread}.
+}
+
+\title{\includegraphics[width=6cm]{\images/title.png}\\The Rewrite}
```

Comment:
Can you translate it straight away? We'll get the metadata out of the way.

Reply:
Should I do it on my branch and submit a pull request ?

Reply:
Just push it to your existing branch so that it's updated here.